packages/mdc-chips/chip-set/foundation.ts

```diff
@@ -110,6 +110,12 @@ export class MDCChipSetFoundation extends MDCFoundation<MDCChipSetAdapter> {
     this.adapter_.removeChipAtIndex(index);
     const maxIndex = this.adapter_.getChipListCount() - 1;
     const nextIndex = Math.min(index, maxIndex);
+
+    // Do not throw error if chip set is empty
```

Comment:
Let's update this to read "Early exit if the next chip index is out of range"

Reply:
how about "Early exit if there are no other chips in the set"? I think it makes it clearer why `nextIndex` would be out of range.

Reply:
Sure, sounds good!